js/helpers/ApiClient.js

```diff
@@ -1,4 +1,12 @@
 export class ApiClient {
+  constructor () {
+    if (ApiClient.instance) {
+      return ApiClient.instance;
+    }
+    ApiClient.instance = this;
+    return this;
+  }
+
   baseUrl = 'http://localhost:3000/api';
```

Comment:
This one should be static field or constants. You don't need it as instance field.